src/extension.js

```diff
@@ -29,6 +30,28 @@ function getModulePath(filePath, moduleName) {
   }
 }
 
+/**
+ * @description Get the global node_modules path
+ * @returns The global node_modules path
+ */
+function getGlobalNodeModulesPath() {
+  // Detect the operating system
+  switch (os.platform()) {
+    case 'win32':
+      // Windows
+      return path.join(process.env.APPDATA, 'npm', 'node_modules');
+    case 'darwin':
+      // macOS
+      return path.join(process.env.PREFIX || '/usr/local', 'lib', 'node_modules');
+    case 'linux':
+      // Linux
+      return path.join(process.env.PREFIX || '/usr', 'lib', 'node_modules');
+    default:
+      // Unsupported platform
+      throw new Error('Unsupported platform');
+  }
+}
+
```

Comment:
The global node_modules location is customizable. For example, with `nvm` the location is something like, `~/.nvm/versions/node/v20.10.0/lib/node_modules`.

It also varies with each package manager. You can check the location though:

npm: `npm root -g`
yarn: `yarn global dir`
pnpm: `pnpm bin -g`